core/src/main/scala/chisel3/Module.scala

```diff
@@ -953,6 +954,14 @@ package experimental {
     localModulePrefix.foreach { prefix =>
       Builder.pushModulePrefix(prefix, localModulePrefixUseSeparator)
     }
+
+    /** Mutable record of what layers were created in this module or its children */
+    private[chisel3] val _layers: LinkedHashSet[Layer] = LinkedHashSet.empty[Layer]
+
+    private[chisel3] def layers: Seq[Layer] = {
+      require(isClosed, "Can't get layers before module is closed")
+      return chisel3.layers.defaultLayers ++ _layers.toSeq
```

Comment:
```suggestion
     chisel3.layers.defaultLayers ++ _layers.toSeq
```

Reply:
Thanks! I'm bouncing between too many languages...

Reply:
Fixed in: aaa462043